### ISSUE TITLE
Video player sometimes keeps buffering for forever when a new video is being played

### ISSUE THREAD

**Issue by AlertSubject** [TeamNewPipe/NewPipe#8145]
### Checklist

- [X] I am able to reproduce the bug with the [latest version](https://github.com/TeamNewPipe/NewPipe/releases/latest).
- [X] I made sure that there are *no existing issues* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have taken the time to fill in all the required details. I understand that the bug report will be dismissed otherwise.
- [X] This issue contains only one bug.
- [X] I have read and understood the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### Affected version

0.22.1

### Steps to reproduce the bug

1. Go to timeline feed
2. Click on a video
3. Go full screen (not sure if required, but I always do that)
4. Video keeps buffering for forever (white spinning circle on black background)

### Expected behavior

The video should start playing within a few seconds of buffering

### Actual behavior

Video keeps buffering for forever unless workaround is being used
Note that the buffering issue doesn't happen in 100% of cases but it does happen rather often. 80%ish for me

workaround:
change the resolution to 720p or something else, can switch back afterwards
minimize app and re-open app
Turn on background playback and then resume video again
Tl;dr: Everything that changes the player type seems to get the stream running again

### Affected Android/Custom ROM version

Android 11 / OxygenOS11

### Affected device model

OnePlus9Pro

**Comment by TiA4f8R** [comment]
I am pretty sure your issue is a duplicate of both #6949 and #7630 issues. Could you try both solutions to see if this issue is improved and/or related?

**Comment by gpdevsoft** [comment]
same issue and occut either in non full screen mode, solution #7630 solve the problem, in my test i have choose 16KB, can you release a new stable versione with this fix or we must wait for next release i mean wich one with many fixes?

**Comment by TiA4f8R** [comment]
No, we will not release a hotfix version, you will have to wait the next standard release to get this fix.

**Comment by gpdevsoft** [comment]
ok thank you for answer, however i have try another workaround for standard release (022.1), changing video and audio format to WebM the buffering time significantly reduce

**Comment by pc00per** [comment]
> ok thank you for answer, however i have try another workaround for standard release (022.1), changing video and audio format to WebM the buffering time significantly reduce

I guess it's mostly exoplayer buffering issue, effecting all formats at least for me.
Just wait for the hotfix, even though the window gap goes too high. Or use [this test debug app](https://github.com/TeamNewPipe/NewPipe/suites/5494503468/artifacts/175647715) for now.
